largest_palindrome_product.js

```diff
@@ -5,18 +5,41 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function(digits) {
   var factor_0 = 0;
   var factor_1 = 0;
   var palindromeNumber = 0;
-
+  let palArr = [];
   // do your work here
+  let f1Arr = [];
```

Comment:
Actually I take back the approval. Instead of having the factors as arrays, try to use numbers instead.